third_party/glfw/BUILD.gn

```diff
@@ -0,0 +1,51 @@
+# Copyright 2016 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
```

Comment:
The one in the root of the repo.  There isn't a LICENSE file in this directory.  The license is contained in a file called COPYING.txt, matching the upstream glfw repository.  This is similar to what we and Chromium do for libpng, etc.